plaidbench/plaidbench/core.py

```diff
@@ -286,8 +286,11 @@ def _inner_run(reports,
 
         # Record stopwatch times
         execution_duration = overrides.get('time', stop_watch.elapsed())
-        tile_exec_per_example = 1e-9 + timef.tot_time_ns / 10.0**9 / params.examples
         exec_per_example = execution_duration / params.examples
+        tile_exec_per_example = exec_per_example
+        if not timef.tot_time_ns == 0:
```

Comment:
Maybe `if timef.tot_time_ns:` (using a truthy expression)

Reply:
Not sure why I wrote it that way.  Must be late.